app/experimenter/experiments/api/v5/forms.py

```diff
@@ -0,0 +1,9 @@
+from django import forms
+
+from experimenter.experiments.models.nimbus import NimbusExperiment
+
+
+class CreateExperimentForm(forms.ModelForm):
+    class Meta:
+        model = NimbusExperiment
+        fields = ("name", "slug", "hypothesis", "public_description")
```

Comment:
OOOOOOOHHHHH it uses FORMS?? Oooooohhh.  Well that's nice, that's a very nice pattern to do validation!  Heck we even have mixins for doing changelogs automatically in forms so we can reuse that.  That's so interesting.